src/Systems/MultiplayerSystem.h

```diff
@@ -79,7 +92,15 @@ class MultiplayerSystem : public System
     void setup(const std::string_view& ip, const std::string_view& port) noexcept;
     void setPlayer(const uint32_t id, const Entity entity);
     void setRoom(const glm::ivec2& room) noexcept;
+    void clearRemoteDungeonUpdates();
     void update(const float deltaTime);
+    void updateMap(const std::map<Entity, sf::Vector2<float>>& enemies,
+                   const std::map<Entity, sf::Vector2<int>>& players);
+    void sendMapDimensions(const std::unordered_map<Entity, ObstacleData>& obstacles);
+    void gatherEnemyAndPlayerPositions();
+    void sendSpawnerPosition(comm::StateUpdate& stateUpdate,
```

Comment:
To wszystko jest private zamiast public

Reply:
Zmienię to na public, ale w sumie nie wiedziałem po co to ma być publiczne, jak będę używał tego tylko w tym systemie.